content/docs/welcome-guide/setup/installing-linux.md

```diff
@@ -105,3 +105,43 @@ Example of launching Project Manager from the shell:
     (Reading database ... 66411 files and directories currently installed.)
     Removing o3de (0.0.0.0) ...
     ```
+
+## Installing O3DE for Linux from a Snap package
+
+{{< note >}}
+The Snap package is experimental and may run into issues on some distros. This has been tested on Ubuntu 20.04 and 22.04 LTS
+{{< /note >}}
+
+1. Download the snap package from the [O3DE download](https://o3de.org/download/#linux) page or the [Snap store](https://snapcraft.io/o3de)
+
+1. Depending on disto, you will need to install `snapd` in order to install the snap package. [Refer to this guide for instructions specific to your distro](https://snapcraft.io/docs/installing-snapd)
+
+1. If the snap package has been downloaded from the O3DE page, use this command to install the package
+   ```shell
+   snap install --classic --dangerous <o3de snap package filename>.snap
+   ```
```

Comment:
```suggestion
   ```shell
   snap install --classic --dangerous <o3de snap package filename>.snap
   ```
   The [`--dangerous` ](https://snapcraft.io/docs/install-modes#heading--dangerous)option is not needed if installed from the snap store.
```

Reply:
This instruction is if the snap package is downloaded form o3debinaries.org, in the event that the snap store approval is not ready by release. It's needed as there will not be any signed hash metadata from the snap store if the package is not published.

Reply:
That said, will remove this flag if we do get the snap published